portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -21,28 +21,36 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet
 {
+    public class LoginState
+        {
+            private Boolean login = false;
+            private String loginUrl;
+        }
+
     @Override
     public void doGet (HttpServletRequest request, HttpServletResponse response) throws IOException
     {   
-        //Sets response type to html. We are writing directly to the page.
-        response.setContentType("text/html");
+        LoginState person = new LoginState();
+
+        //Sets response type to json.
```

Comment:
Comment is kind of unnecessary I think. Your line of code that follows is very clear in its purpose.